portfolio/src/main/webapp/index.html

```diff
@@ -80,8 +83,21 @@ <h2 class="page_headers">Leave a comment!</h2>
             </template>
             <div id="comment-selector-container">
               <form action="/load-comments" method="GET">
-                <label for="number-of-comments"> Number of Comments: </label>
-                <input type="number" name="number-of-comments" id="number-of-comments" max="7" min="0" value="0">
+                <div>
+                  <label for="number-of-comments"> # of Comments: </label>
```

Comment:
nit: I'm not sure ' # ' as a replacement for the word 'number' translates to every culture/language? If someone is looking to translate the page to say Hindi, will the # symbol be meaningful?

Reply:
You make a really good point. I did some research and it seems that in the USA and some countries of Europe it is taken as the meaning of number. I know for sure that in some countries in Central and South America has that meaning as well. Although, I wasn't able to find more information about what it means in other countries. Being this, accessibility first, will be changed back to the word instead of the symbol.